### PR TITLE
Fix the model preview not always rendering parts assigned to row 0

### PR DESCRIPTION
**Pull request by KoraLyn**
Relevant framework PR: https://github.com/TexTools/xivModdingFramework/pull/33

### CODE REVIEW

File: FFXIV_TexTools/Views/ModPack/Wizard/WizardModPackControl.xaml.cs [hunk 1]
```diff
@@ -225,23 +226,30 @@ private void OptionsList_SelectionChanged(object sender, SelectionChangedEventAr
...
+                            bmp.StreamSource = ms;
+                            bmp.CacheOption = BitmapCacheOption.OnLoad;
+                            bmp.EndInit();
+                            bmp.Freeze();
+                        }
 
-                    using (var ms = new MemoryStream())
+                        OptionPreviewImage.Source = bmp;
+                    }
+                    catch(ObjectDisposedException)
```
Comment (KoraLyn) [inline FFXIV_TexTools/Views/ModPack/Wizard/WizardModPackControl.xaml.cs:249]: `Image.IsDisposed` was made private in one of the updates to ImageSharp and instead they opted for throwing a `ObjectDisposedException`  when `IsDisposed` is true as can be seen in their source below. I think this is equivalent to what we had before.

https://github.com/SixLabors/ImageSharp/blob/master/src/ImageSharp/Image.cs#L161

**Comment by Squall-Leonhart** [comment]
Did you intend to change the system buffer and system memory dlls from netstandard to net461?

seems like it won't make a major difference, 461 has fixes for some netstandard 2.0 issues i guess

**Comment by KoraLyn** [comment]
> 
> 
> Did you intend to change the system buffer and system memory dlls from netstandard to net461?
> 
> seems like it won't make a major difference, 461 has fixes for some netstandard 2.0 issues i guess

Uh, wasn't intentional no. I just let NuGet do its thing. Not sure if it matters.

**Comment by Squall-Leonhart** [comment]
It most likely doesn't for textools use case

> ✔️ CONSIDER adding a target for net461 when you're offering a netstandard2.0 target.

>    Using .NET Standard 2.0 from .NET Framework has some issues that were addressed in .NET Framework 4.7.2. You can improve the experience for developers that are still on .NET Framework 4.6.1 - 4.7.1 by offering them a binary that is built for .NET Framework 4.6.1.

**Comment by Squall-Leonhart** [comment]
It seems something weird happened with the nuget update in this case, you'll have a conflict to resolve regardless which way you merge in the prs, so probably just use the netstandard2.0 line when that happens.

edit, my mistake, seems that it did the same on my end for one of the nuget packages that updated system.buffers.

**Pull request merged**